mbedtls/src/lib.rs

```diff
@@ -21,7 +21,7 @@ extern crate bitflags;
 #[macro_use]
 extern crate serde_derive;
 // required explicitly to force inclusion at link time
-#[cfg(target_env = "sgx")]
+#[cfg(all(target_vendor = "fortanix", target_env = "sgx"))]
```

Comment:
Why is fortanix needed here?

Reply:
[rs_libc](https://docs.rs/rs-libc/latest/rs_libc/index.html) is a fortanix crate that is only available in with the fortanix target triple, https://github.com/mobilecoinofficial/rust-mbedtls/blob/feature/environment_target/mbedtls/Cargo.toml#L34.
It provides allocator functions.  We already have another implementation that we use in the enclaves. 

Unfortunately I'm not seeing a way to have `cfg` look at the target triple in its entirety, https://doc.rust-lang.org/reference/conditional-compilation.html, so added the vendor to better exclude when `rs_libc` is used.